CHANGELOG.md

```diff
@@ -0,0 +1,38 @@
+
+6.0.0-beta.5 / YYYY-MM-DD
+==================
+
+## New features
```

Comment:
My idea was to have 3 practical categories:

1. "Here are some new stuff you should check out"
2. "Here are some stuff you need to be aware of"
3. "Here are some stuff you don't need to worry about"

But I'll try and divide it up the way you suggested.